bee-message/src/output/feature_block/mod.rs

```diff
@@ -199,22 +199,57 @@ pub(crate) fn validate_allowed_feature_blocks(
     Ok(())
 }
 
-bitflags! {
-    /// A bitflags-based representation of the set of active feature blocks.
-    pub(crate) struct FeatureBlockFlags: u16 {
-        /// Signals the presence of a [`SenderFeatureBlock`].
-        const SENDER = 1 << SenderFeatureBlock::KIND;
-        /// Signals the presence of a [`IssuerFeatureBlock`].
-        const ISSUER = 1 << IssuerFeatureBlock::KIND;
-        /// Signals the presence of a [`DustDepositReturnFeatureBlock`].
-        const DUST_DEPOSIT_RETURN = 1 << DustDepositReturnFeatureBlock::KIND;
-        /// Signals the presence of a [`TimelockFeatureBlock`].
-        const TIMELOCK = 1 << TimelockFeatureBlock::KIND;
-        /// Signals the presence of a [`ExpirationFeatureBlock`].
-        const EXPIRATION = 1 << ExpirationFeatureBlock::KIND;
-        /// Signals the presence of a [`MetadataFeatureBlock`].
-        const METADATA = 1 << MetadataFeatureBlock::KIND;
-        /// Signals the presence of a [`IndexationFeatureBlock`].
-        const INDEXATION = 1 << IndexationFeatureBlock::KIND;
+macro_rules! create_bitflags {
```

Comment:
Can you put it in the `macro` module ? I am soon splitting the feature blocks in two and I'd need to reuse this in another module.

Reply:
I'll do that. I just first have to figure out how to get the comments to render correctly. Also, if we want to reuse this code, we need to come up with general rustdoc comments.